cmake/podioMacros.cmake

```
@@ -0,0 +1,95 @@
+#---------------------------------------------------------------------------------------------------
+#---REFLEX_GENERATE_DICTIONARY( dictionary headerfiles SELECTION selectionfile OPTIONS opt1 opt2 ...
```

```suggestion
#---PODIO_GENERATE_DICTIONARY( dictionary headerfiles SELECTION selectionfile OPTIONS opt1 opt2 ...
```
This comment refers to `PODIO_GENERATE_DICTIONARY` or should remind the use of `REFLEX...`?

---

Thanks, I modified this line as well. Or did you mean something else?

---

No that's all I meant. Thanks!